src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -248,7 +249,7 @@ protected function _isSelected($key, $selected)
      * @param array|bool|null $disabled The disabled values.
      * @return bool
      */
-    protected function _isDisabled($key, $disabled)
+    protected function _isDisabled(string $key, $disabled): bool
```

Comment:
We could remove the string casts inside this method and `_isSelected`.

Reply:
I already did.

Reply:
:clap: